Jenkinsfile

```diff
@@ -1,15 +1,22 @@
-@Library('socrata-pipeline-library@3.0.0') _
+@Library('socrata-pipeline-library@6.0.0') _
 
 commonPipeline(
     defaultBuildWorker: 'build-worker',
     jobName: 'geo-export',
     language: 'scala',
-    languageVersion: '2.12',
+    languageOptions: [
+	    crossCompile: true,
+        isMultiProjectRepository: false,
```

Comment:
Can be omitted because it defaults to false

Reply:
Removed!